src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -3166,6 +3166,7 @@ def test_aks_automatic_sku(self, resource_group, resource_group_location):
             "AKSHTTPCustomFeatures=Microsoft.ContainerService/EnableAPIServerVnetIntegrationPreview,"
             "AKSHTTPCustomFeatures=Microsoft.ContainerService/SafeguardsPreview,"
             "AKSHTTPCustomFeatures=Microsoft.ContainerService/NRGLockdownPreview,"
+            "AKSHTTPCustomFeatures=Microsoft.ContainerService/AKS-PrometheusAddonPreview,"
```

Comment:
Please mark the case as `@live_only()` as the workspace id is not mocked properly so the test could not be replayed.

Reply:
Marked the case as @live_only() with comments